### DISCUSSION TITLE
REGULATORY Submissions with OSP-Suite

### DISCUSSION THREAD

**Discussion by StephanSchaller** [Open-Systems-Pharmacology/Forum#1135] [Announcement]
Dear Community!

As the OSP Focus Group for Community Engagement and Public Relations (see [Roadmap](https://github.com/Open-Systems-Pharmacology/Roadmap)), we would like to compile a comprehensive list of examples where the OSP Suite has been used in **regulatory submissions** and a whitepaper on the most impactful examples elaborated on in more detail.

For this, we need everyone's help!
Please reach out to us if you have examples you can/want to contribute, either only listed or, if you think of impact, as a co-author to the whitepaper. 

Please reach out to *stephan.schaller@esqlabs.com* for more details.

Have a great week!

**Comment by StephanSchaller** [comment]
Please share this with your networks!

**Reply by arketkar** [reply]
Hi @StephanSchaller and Community!

End of the last year (2021), I submitted one PBPK report for seeking a biowaiver using PK-Sim v10.0.2.57.

The reports were uploaded in Module 5.3.1.3. For that we took a reference from Pg. 48 of 57 of the file from the following link.
https://www.accessdata.fda.gov/drugsatfda_docs/nda/2017/209203Orig1s000ChemR.pdf

The .pksim5 files cleared the FDA submission gateway, without even requiring a zip file.

Hope this helps you and others in the forum!

Regards.
Anant

**Reply by prvmalik** [reply]
Thanks for your help @arketkar 
In January 2022 we submitted two PBPK reports to EMA (CYP3A4, pediatric)
In April 2022 we submitted two PBPK reports to FDA (CYP3A4, CYP2C8). For this we used Module 5.3.5.3.

**Reply by arketkar** [reply]
That's really great @prvmalik ! Thanks for sharing that.

**Reply by StephanSchaller** [reply]
@prvmalik, really great! Thanks for sharing!

**Comment by StephanSchaller** [comment]
Dear Community, this is a reminder to keep this thread up-to-date!

**Reply by ErikssonJ** [reply]
Hi Stephan and others! We are currently looking for PBPK or QSP models developed in PK-Sim/MoBi which has been accepted by PMDA. Do you know of any examples?

**Reply by StephanSchaller** [reply]
Dear @ErikssonJ, yes, there have been (so far three) submissions by MT Members we know of. However, we are still collecting the details and some official public ressources so we can reference these.

**Comment by drmansoor** [comment]
I hope you are keeping well,our research team has been involved in the silico studies using pk sim since the beginning of the system.We are first who got the licence of pk sim in Pakistan. My colleague prof.Dr.Tasneem Ahmed along with my student name Sharib also joined Bayer in Germany to attend the training seminar. We also published a few articles and currently our group started a project related to pk sim. We will appreciate it if you supervise and guide in this regard.
kindly, provide me your email address for further discussion .
Thanks
Best Regards
Prof.Dr.Mansoor Ahmed.
department: pharmaceutical chemistry , university of Karachi Pakistan
